GSoC-2025/brainglobe: Update atlas packaging scripts to atlas API v2 (Swapnaneel Patra).md

```diff
@@ -0,0 +1,99 @@
+        - Update and expand documentation to reflect new usage and features.
+
+    2. Additional **stretch goals** or "if time allows" deliverables
+        - Extend support to additional packaging scripts within the BrainGlobe ecosystem.
+        - Develop conversion utilities for legacy atlas data.
+        - Create example notebooks and tutorials demonstrating the new capabilities.
+
+    3. A detailed **weekly timeline**: when do you plan to do what?
+
+| Week       | Tasks                                                                                                                                       |
+|------------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| **Community Bonding**| Become familiar with the BrainGlobe Atlas API and OpenMINDS SANDS spec; set up development environment; initial discussions with mentors.      |
+| Week 1     | Investigate current packaging scripts; analyze differences between existing format and OpenMINDS SANDS; draft initial design ideas.          |
+| Week 2     | Prepare a detailed design document outlining required code changes and data schema adjustments; start initial code modifications.            |
+| Week 3     | Implement core modifications in the packaging code for OpenMINDS SANDS support; begin writing unit tests.                                     |
+| Week 4     | Adapt one selected packaging script to use the new functionality; continue developing tests and refining the conversion process.             |
+| Week 5     | Integrate mentor feedback; refine code and tests; update documentation draft with examples and usage instructions.                           |
+| Week 6  **Mid-term**     | Ensure more than half of the project is complete; perform integration tests; address any major issues or blockers.              |
+| Week 7     | Implement stretch goals (if feasible); extend test coverage; review and improve code performance and readability.                             |
+| Week 8     | Polish code; perform extensive manual testing with various atlas datasets; update documentation based on test outcomes.                        |
+| Week 9     | Finalize additional features; prepare comprehensive test suites; conduct peer reviews with mentors and community contributors.              |
+| Week 10    | Begin code freeze process; focus on bug fixes, documentation polishing, and ensuring all deliverables meet the OpenMINDS SANDS standard.      |
+| Week 11    | Final refinements; complete any remaining tests; draft the final report and submission documentation.                              |
+| Week 12    | Finalize the project deliverables; ensure all documentation, tests, and code are up-to-date; prepare for final project submission and review. |
+
+I plan to dedicate around 20 hours per week throughout the program.
```

Comment:
Could you clarify what size project you are applying for? This works out at 240 hours, which is neither medium or large.